packages/itwinui-react/src/core/Toast/Toaster.test.tsx

```diff
@@ -19,6 +19,15 @@ import {
 
 const mockOnClick = vi.fn();
 
+// Set reduced motion to not trigger animations since JSDom doesn't mock `animate()`.
+const originalMatchMedia = window.matchMedia;
+beforeAll(() => {
+  window.matchMedia = vi.fn().mockReturnValue({ matches: false });
+});
+afterAll(() => {
+  window.matchMedia = originalMatchMedia;
+});
```

Comment:
Can we use `animate?.()` to get around this issue? Otherwise our consumers will also need to add this mock.